schema/deploy/tables/project.sql

```diff
@@ -5,7 +5,8 @@ begin;
 create table cif.project(
```

Comment:
you could add a
```sql
-- requires: tables/funding_stream
```
comment to have sqitch enforce order ([example in ciip](https://github.com/bcgov/cas-ciip-portal/blob/6aef3cd1833ae44ee3230c0bcc17eb8eea392fe9/schema/deploy/tables/application_revision_status_001.sql#L2))

Reply:
added